jupyterfs/metamanager.py

```diff
@@ -36,8 +35,13 @@
 class MetaManager(AsyncContentsManager):
     copy_pat = re.compile(r"\-Copy\d*\.")
 
-    @default("files_handler_params")
-    def _files_handler_params_default(self):
+    # https://github.com/jpmorganchase/jupyter-fs/issues/175
+    # @default("files_handler_params")
+    # def _files_handler_params_default(self):
+    #     return {"path": self.root_dir}
```

Comment:
Is there a reason to keep this here and not just let the git(hub) history remember it for us?